src/shared/api/urls.ts

```diff
@@ -153,12 +153,19 @@ export function getProxyUrlIfNecessary(url:any) {
     }
 }
 
-export function getOncoKbApiUrl() {
-    let url = AppConfig.serverConfig.oncokb_public_api_url;
-    if (typeof url === 'string') {
-        return buildCBioPortalAPIUrl(`proxy/${trimProtocol(url)}`);
-    } else {
-        return undefined;
+export function getOncoKbProxyApiUrl() {
+    if (AppConfig.serverConfig.oncoKbTokenEnabled) {
+        return buildCBioPortalAPIUrl(`proxy/oncokb`);
+    }
+    // TODO workaround for temporary backward compatibility
+    //  remove this after documenting OncoKB token for all portal instances
+    else {
+        let url = (AppConfig.serverConfig as any).oncokb_public_api_url;
```

Comment:
need to cast as any since `oncokb_public_api_url` is no longer part of the `serverConfig` definition